docs/components/modeler/web-modeler/play-your-process.md

```diff
@@ -107,40 +149,48 @@ Both offer monitoring of a single process instance, its variables and path, inci
 
 ## Limitations and availability
 
-:::note
-Play is being rebuilt and progressively rolled out to more users. This section explains why you might not see the **Play** tab.
-:::
+Play is being rebuilt and progressively rolled out to more users. This section explains why you might not see the **Play** tab, and any additional limitations.
 
-For Camunda 8 SaaS, Play is available to all Web Modeler users with editor or admin permissions within a project.
-Enterprise users need an admin to enable Play by opting in to [alpha features](/components/console/manage-organization/enable-alpha-features.md).
+For more information about terms, refer to our [licensing and terms page](https://legal.camunda.com/licensing-and-other-legal-terms#c8-saas-trial-edition-and-free-tier-edition-terms).
+
+### Camunda 8 SaaS
 
-For Self-Managed, Play is controlled by the **PLAY_ENABLED** flag. It is `true` by default for the Docker distribution for development and `false` by default on the Kubernetes distribution for production use.
+In Camunda 8 SaaS, Play is available to all Web Modeler users with editor or admin permissions within a project.
+Enterprise users need an admin to enable Play by opting in to [alpha features](/components/console/manage-organization/enable-alpha-features.md).
 
-Play uses Zeebe 8.2. Any BPMN elements unavailable in Zeebe 8.2, such as signal events, will not be available in Play.
+### Camunda 8 Self-Managed
 
 :::note
-[Inbound Connectors](/components/connectors/connector-types.md#inbound-connectors) and [Connectors in hybrid mode](/guides/use-connectors-in-hybrid-mode.md) do not connect to external systems and must be completed manually.
-[Start events with forms](/components/modeler/web-modeler/advanced-modeling/publish-public-processes.md#embed-form-in-start-event) will be completed without the form being shown.
-[Decision table rule](/components/modeler/dmn/decision-table-rule.md) evaluations are not viewable. However, they can be inferred from the output variable.
+To use Play with Docker, ensure OAuth is enabled for your configured components. The `docker-compose-core.yaml` file in the Camunda [platform repository](https://github.com/camunda/camunda-platform) does not provide authentication, and cannot be used with Play.
 :::
 
-There are some bugs related to Play’s architecture. These will be resolved when Play is integrated with Camunda development clusters.
+In Self-Managed, Play is controlled by the `PLAY_ENABLED` [configuration property](/self-managed/modeler/web-modeler/configuration/configuration.md#feature-flags) in Web Modeler. This is `true` by default for the 8.6 Docker and Kubernetes distributions, and can be accessed as an alpha feature in 8.6.0-alpha5.
```

Comment:
Self-managed does not have the concept of alpha features in the same way as SaaS, but the alpha5 release is still relevant for SM

Reply:
Correct me, but I think the reference to alpha can be stripped from this? It won't be alpha anymore after we release 8.6 and this is the /next docs page, right?